weechat_otr_test/test_command_finish.py

```diff
@@ -21,23 +21,23 @@ def after_setup(self):
         weechat_otr.ACCOUNTS['nick@server'] = account
 
     def test_finish_buffer(self):
-        weechat_otr.command_cb(None, 'server_nick2_buffer', 'finish')
+        self.call_command_cb(None, 'server_nick2_buffer', 'finish')
```

Comment:
Hmm... maybe you're right. I'm not sure.

It feels wrong to have to decide whether or not to call `to_str` based on the contents of the string. And adding it to every string is very verbose. But adding our own `assert*` functions with magic string conversion is probably just as bad. So yeah, I'll do that after #134.